README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+
+# hw01
+
+My name is **Bingxin** (Janssie) Zhu. In this course, I  want to 
+
+* Learn how to code using R
+* Conduct data anaglsis using R 
```

Comment:
Good job using bold text and a list!
I think a header would have looked great at the beginning.